admin/app/components/solidus_admin/products/index/component.html.erb

```diff
@@ -1,8 +1,18 @@
-<div class="<%= stimulus_id %>">
-  <header class="mb-6">
+<div class="<%= stimulus_id %> px-4">
+  <header class="py-6 flex items-center">
     <h1 class="body-title">
       <%= Spree::Product.model_name.human.pluralize %>
     </h1>
+
+    <div class="ml-auto">
+      <%= render component("ui/button").new(
```

Comment:
Do we need to rebase on top of https://github.com/solidusio/solidus/pull/5254?

Reply:
I have updated the products/index component @waiting-for-dev 👍